settings.go

```diff
@@ -47,6 +47,7 @@ type config struct {
 	CABundlePath       string `split_words:"true" envconfig:"CA_BUNDLE"`
 	SessionStorePath   string `split_words:"true" default:"/var/lib/authservice/data.db"`
 	SessionMaxAge      int    `split_words:"true" default:"86400"`
+	CookieSameSite     string `split_words:"true" default:"Strict"`
```

Comment:
Same here, can you make `Lax` the default?
Also, can you use `SessionSameSite`, same as the setting?

Reply:
done